src/PureSMT/Process.hs

```diff
@@ -1,74 +1,67 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE TemplateHaskell #-}
+
 module PureSMT.Process where
 
 import Control.Monad
-import Data.Maybe (fromMaybe)
-import Data.String (fromString)
+import qualified Data.ByteString.Char8 as BS
+import Foreign.Ptr (Ptr)
+import qualified Language.C.Inline as C hiding (block, exp)
+import qualified Language.C.Inline.Unsafe as C
```

Comment:
I would import this as `CU`, as the documentation of `Language.C.Inline.Unsafe` suggests. Then it isn't necessary to hide `block` and `exp` in the import of `Language.C.Inline`.